### PR TITLE
avatar and badge component merge ready

### PR DESCRIPTION
**Pull request by b0llu**
both the components are merge ready from my side if any suggestion please give feedback.

### CODE REVIEW

File: Components/Avatar/Avatar.css [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    margin: 0 1rem;
+    object-fit: cover;
+    height: 9rem;
+    width: 9rem;
+    border: 2px solid var(--white);
+}
+
+
+/* -------------- sized avatar ---------- */
+.la {
```
Comment (agauravdev) [inline Components/Avatar/Avatar.css]: 1. lg would be more appropriate than la.. 🤔 
2. A more general approach would be to use avatar-lg.  Only 'la' might create confusion. If you want to keep it like this, then you can try creating utility libraries 🤔  which would work on major components.

File: Components/Avatar/Avatar.css [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    width: 9rem;
+}
+
+.sm {
+    height: 5rem;
+    width: 5rem;
+}
+
+/* ---------------- square avatar--------- */
+.square {
```
Comment (agauravdev) [inline Components/Avatar/Avatar.css]: Same as above. Try looking into BEM naming convention 🤔

**Review by agauravdev — CHANGES_REQUESTED** [review]
Just the naming for now. :D

File: Components/Avatar/Avatar.css (hunk 1 above)
Comment (b0llu) [reply Components/Avatar/Avatar.css]: I have changed the name to avatar-lg and respectively other also

File: Components/Avatar/Avatar.css (hunk 2 above)
Comment (b0llu) [reply Components/Avatar/Avatar.css]: changed the class name to sq-border as I felt it described what it does better

**Comment by rushi-173** [comment]
Everything looks fine for now :)

**Comment by vashnavichauhan1825** [comment]
clean code :)

**Pull request merged**